code/game/objects/items/tools/shovel_tools.dm

```diff
@@ -143,25 +143,31 @@
 // Entrenching tool.
 /obj/item/tool/shovel/etool
 	name = "entrenching tool"
-	desc = "Used to dig holes and bash heads in. Folds in to fit in small spaces."
+	desc = "Used to dig holes and bash heads in. Folds in to fit in small spaces. Use a machete on it to sharpen it."
 	icon = 'icons/Marine/marine-items.dmi'
 	icon_state = "etool"
 	force = 30
 	throwforce = 2
 	item_state = "crowbar"
+	hitsound = "swing_hit"
 	w_class = WEIGHT_CLASS_BULKY //three for unfolded, 3 for folded. This should keep it outside backpacks until its folded, made it 3 because 2 lets you fit in pockets appearntly.
 	dirt_overlay = "etool_overlay"
 	dirt_amt_per_dig = 5
 	shovelspeed = 20
+	var/sharpened = FALSE
```

Comment:
Items already have a sharp var. I could have told you earlier, but i forgot. Use it